src/staticanalysis/bot/static_analysis_bot/__init__.py

```diff
@@ -175,27 +108,6 @@ def as_phabricator_lint(self):
         '''
         raise NotImplementedError
 
-    def is_third_party(self):
```

Comment:
Did we move this check in the tasks on try? I assume yes, but want to be sure :P

Reply:
Only clang-format has it... i'm restoring this

Reply:
Restored in 72cf2d4adee8ff6310b89f7bf34df9cb97518875